digest/digest_test.go

```diff
@@ -14,21 +12,6 @@ func TestParseDigest(t *testing.T) {
 		hex       string
 	}{
 		{
-			input:     "tarsum+sha256:e58fcf7418d4390dec8e8fb69d88c06ec07039d651fedd3aa72af9972e7d046b",
-			algorithm: "tarsum+sha256",
```

Comment:
I'm talking about https://github.com/docker/distribution/blob/master/digest/digest.go#L118. The `Validate` function only allowed this notation for tarsum. We could change it to support it with a `hash.Hash` algorithm, but IMHO unless we add support for something useful that uses this notation, it's correct for `Validate` to reject it.